base-template.sh

```diff
@@ -45,6 +45,30 @@ are_githooks_disabled() {
     return 1
 }
 
+#####################################################
+# Sets the ${INSTALL_DIR} variable
+
+# Returns: 0 if success, 1 otherwise
+#####################################################
+load_install_dir() {
+
+    INSTALL_DIR=$(git config --global --get githooks.installDir)
+
+    #shellcheck disable=SC2181
+    if [ $? -ne 0 ]; then
+        # install dir not defined, use default
+        INSTALL_DIR=~/".githooks"
+    elif [ ! -d "$INSTALL_DIR" ]; then
+        echo "! Githooks installation is corrupt! " >&2
+        echo "  Install directory \`${INSTALL_DIR}\` is missing." >&2
+        INSTALL_DIR=~/".githooks"
+        echo "  Fallback to default directory \`${INSTALL_DIR}\`" >&2
+        echo "  Please run the Githooks install script again to fix it." >&2
+    fi
+
+    return 0
```

Comment:
Nit: you can omit this line, it doesn't change anything.

Reply:
ah ok, wasnt sure

Reply:
isn't it so that omitting it means the exit code is the last statement hm...which anyway should not fail

Reply:
Oh, you're probably right. Regardless, this function now can't fail anything and its exit code is not checked, so no need to `return`